sdk/src/wallet/operations/transaction/high_level/minting/create_native_token.rs

```diff
@@ -134,15 +134,20 @@ where
         log::debug!("[TRANSACTION] create_native_token");
         let storage_score_params = self.client().get_storage_score_parameters().await?;
 
-        let (account_id, account_output) = self
+        let (account_id, account_output_data) = self
             .get_account_output(params.account_id)
             .await
             .ok_or_else(|| crate::wallet::Error::MintingFailed("Missing account output".to_string()))?;
 
-        if let Output::Account(account_output) = &account_output.output {
-            // Create the new account output with the same feature blocks, just updated foundry_counter.
+        if let Output::Account(account_output) = &account_output_data.output {
+            // Create the new account output with the same features, just updated mana and foundry_counter.
             let new_account_output_builder = AccountOutputBuilder::from(account_output)
                 .with_account_id(account_id)
+                .with_mana(account_output_data.output.available_mana(
+                    &self.client().get_protocol_parameters().await?,
```

Comment:
I would avoid this 2nd request and instead change line L135 to get the protocol parameters once.

Reply:
There are no 2nd request as it's cached but I've changed it anyway.